C/include/c4DatabaseTypes.h

```diff
@@ -37,6 +37,7 @@ typedef C4_OPTIONS(uint32_t, C4DatabaseFlags) {
     kC4DB_VersionVectors= 0x08, ///< Upgrade DB to version vectors instead of rev trees [EXPERIMENTAL]
     kC4DB_NoUpgrade     = 0x20, ///< Disable upgrading an older-version database
     kC4DB_NonObservable = 0x40, ///< Disable database/collection observers, for slightly faster writes
+    kC4DB_DiskSyncFull  = 0x80, ///< Write to disk after each transaction
```

Comment:
Maybe only SyncFull or FullSync?

Reply:
The comment is also probably misleading.  All transactions write to the disk after they finish successfully, but the full sync also introduces a manual flush.

Reply:
That was my choice in the first attempt, but afterwards, I feel "disk" makes it clearer. It is about the sync between the disk and memory.

Reply:
"Flush to disk after each transaction" ?
@borrrden